BpmChecker.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Vital_Checks
+{
+    //This class is responsible to check bpm vital is okay or not.
+    class BpmChecker
+    {
+         public static bool isBpmOkay(float bpm) {
+

```

Comment:
The pattern of the code here is very similar to RespRateChecker and almost similar to SPO2 check as well.
Can we abstract it by having a generic 'RangeCheck' class that takes the limits and checks against them? It just needs to compare and return true or false. We can have three objects of that class having different limits.

Overall a good split of files 👍